js/script.js

```diff
@@ -0,0 +1,43 @@
+
+const userInput = document.querySelector(".searchInputBox");
```

Comment:
this file sohuld be deleted if it's not being in use. Dont worry, the code is saved inside git.